technische-universitat-dresden-kunstgeschichte-note.csl

```diff
@@ -15,16 +15,17 @@
     <category citation-format="note"/>
     <category field="humanities"/>
     <summary>Zitierstil entsprechend den Vorgaben des Fachbereichs Kunstgeschichte, Technische Universität Dresden.</summary>
-    <updated>2016-12-04T11:18:53+00:00</updated>
+    <updated>2016-12-07T12:14:31+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="de">
     <terms>
-      <term name="accessed">Stand</term>
+      <term name="accessed"></term>
```

Comment:
Well, you are right, @rmzelle, even though the term's emptiness was intentional, it's not elegant and should be deleted.